.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches:
+      # semantic-release valid branches
+      - '+([0-9])?(.{+([0-9]),x}).x'
+      - 'main'
+      - 'next'
+      - 'next-major'
+      - 'beta'
+      - 'alpha'
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  code_validation:
+    name: Code Validation
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Use Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          useLockFile: false
```

Comment:
I think it is not necessary to use an extra GH action here, while a simple 

```yaml
name: Install dependencies
run: npm install
```

should do the trick

Reply:
This npm-install action will cache stuff related to npm properly :)

Reply:
ohh nice